src/Common/src/System/SR.cs

```diff
@@ -16,7 +15,6 @@ private static ResourceManager ResourceManager
 
         // This method is used to decide if we need to append the exception message parameters to the message when calling SR.Format.
         // by default it returns false.
-        [MethodImpl(MethodImplOptions.NoInlining)]
```

Comment:
Please don't remove this. removing it will have the code optimized and we used to override this method in the .Net Native toolchain. 

---
In reply to: [111625461](https://github.com/dotnet/corefx/pull/18386#discussion_r111625461) [](ancestors = 111625461)